group_vars/matrix_servers

```diff
@@ -1090,6 +1096,51 @@ matrix_mautrix_signal_daemon_container_image_self_build: "{{ matrix_architecture
 #
 ######################################################################
 
+######################################################################
+#
+# matrix-bridge-mautrix-signalgo
+#
+######################################################################
+
+# We don't enable bridges by default.
+matrix_mautrix_signalgo_enabled: false
+
+matrix_mautrix_signalgo_container_image_self_build: "{{ matrix_architecture not in ['arm64', 'amd64'] }}"
+
+matrix_mautrix_signalgo_systemd_required_services_list: |
+  {{
+    ['docker.service']
+    +
+    ['matrix-' + matrix_homeserver_implementation + '.service']
+    +
+    ([devture_postgres_identifier ~ '.service'] if devture_postgres_enabled else [])
+    +
+    (['matrix-nginx-proxy.service'] if matrix_nginx_proxy_enabled else [])
+  }}
+
+matrix_mautrix_signalgo_appservice_token: "{{ '%s' | format(matrix_homeserver_generic_secret_key) | password_hash('sha512', 'sigo.as.token', rounds=655555) | to_uuid }}"
+
+matrix_mautrix_signalgo_homeserver_token: "{{ '%s' | format(matrix_homeserver_generic_secret_key) | password_hash('sha512', 'sigo.hs.token', rounds=655555) | to_uuid }}"
+
+matrix_mautrix_signalgo_login_shared_secret: "{{ matrix_synapse_ext_password_provider_shared_secret_auth_shared_secret if matrix_synapse_ext_password_provider_shared_secret_auth_enabled else '' }}"
+
+# People using an external Prometheus server will need to toggle all of these to be able to consume metrics remotely:
+# - `matrix_mautrix_signalgo_metrics_enabled`
+# - `matrix_mautrix_signalgo_proxying_metrics_enabled`
+# - `matrix_nginx_proxy_proxy_matrix_metrics_enabled`
+matrix_mautrix_signalgo_metrics_enabled: "{{ prometheus_enabled }}"
+
+# Postgres is the default, except if not using internal Postgres server
+matrix_mautrix_signalgo_database_engine: "{{ 'postgres' if devture_postgres_enabled else 'sqlite' }}"
```

Comment:
This forces the engine to be postgres if managed postgres is present, which sounds reasonable. But maybe should respect [these lines](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/173286470cc44523df878451d8efb1772afef75a/roles/custom/matrix-bridge-mautrix-signalgo/defaults/main.yml#L68-L74)?